core/templates/components/ck-editor-helpers/ck-editor-copy-content.service.spec.ts

```diff
@@ -127,7 +127,8 @@ describe('Ck editor copy content service', () => {
         startupData: {
           alt: '',
           caption: 'Banana',
-          filepath: 'img_20200630_114637_c2ek92uvb8_height_326_width_490.png'
+          filepath: 'img_20200630_114637_c2ek92uvb8_height_326_width_490.png',
```

Comment:
As Vojta pointed out, should we have a test to check that the content is not copied when the modal is canceled?

Reply:
I have added an e2e test for this in the contributorDashboard suite.